checkstyle-tester/checks-nonjavadoc-error.xml

```diff
@@ -350,6 +350,7 @@
     <module name="UnnecessarySemicolonAfterTypeMemberDeclaration"/>
     <module name="UnnecessarySemicolonInEnumeration"/>
     <module name="UnnecessarySemicolonInTryWithResources"/>
+    <module name="UnusedCatchParameterShouldBeUnname"/>
```

Comment:
@mahfouz72 Last I looked, we named the check `UnusedCatchParameterShouldBeUnnamed` (missing `d`).

Reply:
done